pages/sortinglist/[slug].js

```diff
@@ -117,51 +117,54 @@ export default function DynamicForm() {
   const currentTree = treeData.find((element) => element.slug === treeFromSlug);
   const attributes = currentTree.attributes;
 
+  let inputNumber = 1;
+
+  const renderInputField = (label) => {
+    const inputId = `input$(inputNumber)`;
+    inputNumber++;
+    return (
+      <p>
+        <label htmlFor={inputId}>{label}</label>
+        <input type="text" name={inputId} id={inputId} />
```

Comment:
I suggest to use `label` as name. 
```suggestion
        <input type="text" name={label} id={inputId} />
```
its better readable later when reading the inputs.